file_paths.md

```diff
@@ -204,6 +217,20 @@ user or group.
             run                               # :rundir                      same
                 agent.pid                     # :pidfile
 
+    C:\ProgramData\PuppetLabs\pxp-agent *
+        etc *
+            pxp-agent.cfg                     # pxp-agent configuration file
+            modules.d *                       # directory where pxp-agent modules are stored
+                puppet.cfg *                  # ../../var/modules/puppet configuration
+        var *
+            log
+                pxp-agent.log                 # enabled by default
+            modules *
```

Comment:
`modules.d` contains configuration files whereas `modules` the actual scripts. 
@kylog, below, suggests to use the same name for both directories.

My intent was to let scripts and config files to reside on different directory trees, since we have different locations. But I have no strong opinion.

Should scripts and configuration files reside in the same location?
What are the appropriate location(s)?